linkerd/app/src/env.rs

```diff
@@ -266,6 +266,9 @@ const ENV_OUTBOUND_HTTP1_CONNECTION_POOL_IDLE_TIMEOUT: &str =
 
 const ENV_SHUTDOWN_GRACE_PERIOD: &str = "LINKERD2_PROXY_SHUTDOWN_GRACE_PERIOD";
 
+const ENV_POD_UID: &str = "_pod_uid";
+const ENV_POD_CONTAINER_NAME: &str = "_pod_containerName";
```

Comment:
the case here is mildly surprising to me, but that's an implementation detail that i'll consider out of scope for a pr in the proxy.